app/src/main/res/drawable/bg_splash.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item android:drawable="@color/colorPrimary" />
+
+  <!--<item android:drawable="@drawable/ic_oppia_logo_white"-->
```

Comment:
Why is this commented out? Shouldn't we be using the logo?

Reply:
made changes to this file. I have added splash_page as background image